src/client/app/access-checker/admin-cache-entries.component.ts

```diff
@@ -38,8 +38,8 @@ export class AdminCacheEntriesComponent {
 		this.alertService.clearAllAlerts();
 
 		this.pagingOpts = new PagingOptions();
-		this.pagingOpts.sortField = this.sortOpts['key'].sortField;
-		this.pagingOpts.sortDir = this.sortOpts['key'].sortDir;
+		this.pagingOpts.sortField = this.sortOpts.key.sortField;
```

Comment:
This will introduce compiler errors. These statements were deliberately using string literal accessor notation.